src/Symfony/Component/Console/Input/InputInterface.php

```diff
@@ -147,4 +144,11 @@ public function isInteractive(): bool;
      * @return void
      */
     public function setInteractive(bool $interactive);
+
+    /**
+     * Returns a stringified representation of the args passed to the command.
+     *
+     * InputArguments MUST be escaped as well as the InputOption values passed to the command.
+     */
+    public function __toString();
```

Comment:
```suggestion
    public function __toString(): string;
```

can this be added directly? Or it must be defined in v6 before?

Reply:
Makes zero difference, that's strictly the same for the engine

Reply:
Oki copy

Reply:
I would still add it for consistency (we've also added it to all other `__toString()` definitions in the codebase)